modules/test/playwright/tests/export-import-web/main/stagingUseCase.spec.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+import {createReadStream, readdirSync, statSync} from 'fs';
+import path from 'path';
+
+import {applicationsMenuPageTest} from '../../../fixtures/applicationsMenuPageTest';
+import {dataApiHelpersTest} from '../../../fixtures/dataApiHelpersTest';
+import {featureFlagsTest} from '../../../fixtures/featureFlagsTest';
+import {loginTest} from '../../../fixtures/loginTest';
+import getRandomString from '../../../utils/getRandomString';
+import getBasicWebContentStructureId from '../../../utils/structured-content/getBasicWebContentStructureId';
+import {checkFolderInZip} from '../../../utils/zip';
+import {exportImportConfig} from './export_import.config';
+import {stagingConfigurationPageTest} from './fixtures/stagingConfigurationPageTest';
+import {stagingPageTest} from './fixtures/stagingPageTest';
+
+export const test = mergeTests(
+	applicationsMenuPageTest,
+	dataApiHelpersTest,
+	featureFlagsTest({
+		'LPD-35914': {enabled: true, system: true},
+	}),
+	loginTest(),
+	stagingPageTest,
+	stagingConfigurationPageTest
+);
+
+async function _unzipAndCheckFolder(
+	tempDir: string,
+	folderName: string = 'adaptive-media'
+): Promise<boolean | null> {
+	const files = readdirSync(tempDir)
+		.filter((file) => file.endsWith('.lar'))
+		.map((file) => ({
+			file,
+			time: statSync(path.join(tempDir, file)).mtime.getTime(),
+		}));
+
+	if (!files.length) {
+		return null;
+	}
+
+	// Sort files by most recent modification time
+
+	files.sort((a, b) => b.time - a.time);
```

Comment:
Also just as a question, could the `.sort` be done chained as `.filter` and `.map` are?
Interesting solution! :smile: